src/Glue.elm

```diff
@@ -121,6 +121,29 @@ init (Glue { msg }) ( subModel, subCmd ) ( fc, cmd ) =
     ( fc subModel, Cmd.batch [ cmd, Cmd.map msg subCmd ] )
 
 
+{-| Similar to [`update`](#update) but using custom function.
```

Comment:
this is `update` fn but docs for `updateWithTrigger`

Reply:
it whole needs to be refreshed.... And then readme... documentation is design smell :D